qa/tasks/cephfs/kernel_mount.py

```diff
@@ -155,7 +155,7 @@ def umount(self, force=False):
         self.mounted = False
         self.cleanup()
 
-    def umount_wait(self, force=False, require_clean=False, timeout=900):
+    def umount_wait(self, force=False, require_clean=False, timeout=300):
```

Comment:
Still `timeout=(15*60)` is used in below code anyway? Also there is a [UMOUNT_TIMEOUT](https://github.com/ceph/ceph/blob/673857be3e8983df3cc2100c1e95de03b56fe50b/qa/tasks/cephfs/kernel_mount.py#L18), which was never used.

Reply:
Maybe you should switch to use the `UMOUNT_TIMEOUT` instead in all the places.

Reply:
Done, thanks!

Reply:
Need to fix here [1] too in `umount_wait()`

[1] https://github.com/ceph/ceph/blob/master/qa/tasks/cephfs/kernel_mount.py#L176